uwblueprint/childrens-aid-society
# Kevin y/implement put request for intake and add intake fields

## Notion ticket link

[Implement PUT request for Intake and add intake fields](https://www.notion.so/uwblueprintexecs/134e1f159caa4f73939d11a700c325ad?v=39cf7d5a98384eb9af2bff2265d0ac8a&p=0e04e01782094e19bbc29464ad432007&pm=s)



## Implementation description
* Added update_intake function that takes an updated data object and id to update some fields of database entry with that id, and a route to provide the data.



## Steps to test
1. send put request with json object in body with fields intake_status, lead_access_worker_name, intake_meeting_notes, and should update the database



## What should reviewers focus on?
* 


## Checklist
